quickwit/quickwit-serve/src/index_api/rest_handler.rs

```diff
@@ -1640,28 +1633,6 @@ mod tests {
         assert!(indexes.is_empty());
     }
 
-    #[tokio::test]
-    async fn test_create_file_source_returns_403() {
-        let metastore = metastore_for_test();
-        let index_service = IndexService::new(metastore.clone(), StorageResolver::unconfigured());
-        let mut node_config = NodeConfig::for_test();
-        node_config.default_index_root_uri = Uri::for_test("file:///default-index-root-uri");
```

Comment:
can we get a test checking that a uri targetting a local file is forbidden?

(I see you added stdin)

Reply:
I removed that validation as stated [here](https://github.com/quickwit-oss/quickwit/pull/5148#issuecomment-2210625199). But I actually agree it's better not to have this kind of side effects, so I'll add the check back.